modules/platform/nuxeo-platform-video/src/test/java/org/nuxeo/ecm/platform/video/tools/TestVideoToolsOperations.java

```diff
@@ -301,9 +299,9 @@ protected DocumentModel createWatermarkDocument(File input) {
                 "Picture");
         pictureDoc.setPropertyValue("dc:title", input.getName());
         pictureDoc.setPropertyValue("file:content", new FileBlob(input));
+        pictureDoc.putContextData(DISABLE_PICTURE_VIEWS_GENERATION_LISTENER, true);
```

Comment:
why in the context of this refactoring, maybe add a comment?

Reply:
I think this is due to the `nextTransaction` call which makes the _pictureViewsGenerationListener_ running, but failing later due to missing bundle (action framework).

This doesn't prevent the test to succeed but produces some NPE in logs.

I will add a comment such as _not needed_.